libs/sdk-core/src/breez_services.rs

```diff
@@ -339,7 +354,10 @@ impl BreezServices {
                 )?;
 
                 Ok(LnUrlPayResult::EndpointSuccess {
-                    data: maybe_sa_processed,
+                    data: LnUrlSuccessData {
+                        payment_hash: details.payment_hash.clone(),
+                        data: maybe_sa_processed,
```

Comment:
Can I propose we change `data` to `success_action` to avoid `data.data`?

Reply:
Would make sense IMO.